packages/admin-ui/src/components/LinkDocs.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import newTabProps from "utils/newTabProps";
+import { FaReadme } from "react-icons/fa";
```

Comment:
Not a big fan of the FaReadme icon, looks like a butterfly to me haha. I would keep it simple and use an icon of info or help similar to (i) or (?)